api.md

```diff
@@ -48,2669 +48,71 @@ The Tailscale API does not currently support pagination. All results are returne
 
 # APIs
 
-**[Device](#device)**
+**[Device](./publicapi/device.md#device)**
 
-- Get a device: [`GET /api/v2/device/{deviceid}`](#get-device)
-- Delete a device: [`DELETE /api/v2/device/{deviceID}`](#delete-device)
-- Expire device key: [`POST /api/v2/device/{deviceID}/expire`](#expire-device-key)
+- Get a device: [`GET /api/v2/device/{deviceid}`](./publicapi/device.md#get-device)
```

Comment:
Thoughts on adding links to each of these rows that match the previous headers down below?

E.g.
```
<a name="device-get"></a>
```

That way if anyone had a link to a previous header, it would at least take them to the item in the table of contents where they should click through to the docs.